airbyte-integrations/connectors/source-amplitude/source_amplitude/manifest.yaml

```diff
@@ -0,0 +1,171 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer:
+        - "{{ options.get('data_field') }}"
+  requester:
+    type: HttpRequester
+    url_base: "https://{{'analytics.eu.' if config['data_region'] == 'EU Residency Server' else '' }}amplitude.com/api"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['api_key'] }}"
+      password: "{{ config['secret_key'] }}"
+    error_handler:
+      type: DefaultErrorHandler
+      response_filters:
+        - http_codes: [ 400 ]
+          action: IGNORE
+          error_message: The file size of the exported data is too large. Shorten the time ranges and try again. The limit size is 4GB.
+        - http_codes: [ 404 ]
+          action: IGNORE
+          error_message: No data collected
+        - http_codes: [ 504 ]
+          action: IGNORE
+          error_message: The amount of data is large causing a timeout. For large amounts of data, the Amazon S3 destination is recommended.
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "*ref(definitions.selector)"
```

Comment:
Using the latest version of the airbyte-cdk, `$ref` now have the format `#/definitions/selector`

Reply:
updated